tests/PHPUnit/Integration/ReleaseCheckListTest.php

```diff
@@ -621,7 +621,7 @@ public function testTotalPiwikFilesSizeIsWithinReasonnableSize()
             // Don't run the test on local dev machine, as we may have other files (not in GIT) that would fail this test
             $this->markTestSkipped("Skipped this test on local dev environment.");
         }
-        $maximumTotalFilesizesExpectedInMb = 62;
+        $maximumTotalFilesizesExpectedInMb = 63;
```

Comment:
_Using a random line to get a resolvable thread._

The TagManager branch needs an update (and a new submodule update here). By now the linked branch is outdated and merging the update would revert several changes until the submodule gets updated again. Would also solve some confusion around the failing UI test that is only indirectly related to the outdated branch.

Reply:
TagManager branch rebased, submodule updated here, thanks for pointing that out.